.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+- [ ] **Submitter**: Include the JIRA issue number in the PR description
```

Comment:
I like how in GSA-unstable they first have a section for a Description (header) and then a section for a Checklist where this part would all go. Example: https://github.com/broadinstitute/gsa-unstable/pull/1574